apps/frontend/src/libs/components/avatar/styles.module.css

```
@@ -10,7 +10,24 @@
 	border-radius: 50%;
 }
 
+.no-image {
+	border: 2px solid var(--color-border-primary);
+	transition: all 0.3s ease;
```

let's define the property for transition
```suggestion
	transition: border 0.3s ease;
```